cli/cmd/upgrade.go

```diff
@@ -16,6 +16,9 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const upgradeURL = "https://linkerd.io/upgrade/"
+const upgradeInstructions = "\nYou're on your way to upgrading Linkerd! Visit this URL for installation instructions: " + upgradeURL + "\n"
```

Comment:
What do you think about splitting this into two lines and adding some color? As it stands, the output kinda melts together on my screen.

![Uploading Screenshot 2019-04-03 08.32.50.png…]()

Also, how does this sound - `Visit this URL for further instructions`?

Reply:
Great idea. I used the existing color characters (√, ×) - let me know what you think.